core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -338,4 +339,12 @@ private BufferedReader getReaderForContainerPort80(GenericContainer container) {
             return new BufferedReader(new InputStreamReader(socket.getInputStream()));
         });
     }
+
+    @Test
+    public void addExposedPortAfterWithExposedPortsTest() {
+        redis.addExposedPort(8987);
+        assertThat("Both ports should be exposed", redis.getExposedPorts().size(), equalTo(2));
```

Comment:
I would love to use [containsInAnyOrder](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/collection/IsIterableContainingInAnyOrder.html#containsInAnyOrder(java.util.Collection)) instead of this series of assertions, but it is not available in dependencies and I don't feel like I can add it just for this case.